### PR TITLE
Test for #6067

### PR DESCRIPTION
**Pull request by AndriySvyryd**
@ajcvickers

### CODE REVIEW

File: test/Microsoft.EntityFrameworkCore.Tests/ChangeTracking/Internal/FixupTest.cs [hunk 1]
```diff
@@ -3194,6 +3194,43 @@ public void Add_principal_but_not_dependent_one_to_one_no_navs_FK_set_no_navs_se
             }
         }
 
+        //Issue #6067
+        //[Fact]
```
Comment (anpete) [inline test/Microsoft.EntityFrameworkCore.Tests/ChangeTracking/Internal/FixupTest.cs]: Need [Fact]?

Comment (AndriySvyryd) [inline test/Microsoft.EntityFrameworkCore.Tests/ChangeTracking/Internal/FixupTest.cs]: Would break the build as the test fails

Comment (anpete) [inline test/Microsoft.EntityFrameworkCore.Tests/ChangeTracking/Internal/FixupTest.cs]: Ah, mistakenly thought we were verifying the issue was fixed :relaxed:

**Comment by ajcvickers** [comment]
:shipit:

**Pull request merged**